subsys/bluetooth/mesh/friend.c

```diff
@@ -1239,6 +1380,10 @@ static bool friend_lpn_matches(struct bt_mesh_friend *frnd, u16_t net_idx,
 		return is_lpn_unicast(frnd, addr);
 	}
 
+	if (addr == BT_MESH_ADDR_ALL_NODES) {
+		return true;
+	}
```

Comment:
Side note: I think we have the same bug in `client_filter_match()` in `mesh/proxy.c`